src/app/authentication/components/form-login/form-login.component.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { FormLoginComponent } from './form-login.component';
+
+describe('FormLoginComponent', () => {
+  let component: FormLoginComponent;
+  let fixture: ComponentFixture<FormLoginComponent>;
+  let baseElement: any;
+
+  const SELECTORS = {
+    emailField: '.form-login input[type="email"]',
+    passworField: '.form-login input[type="password"]',
+    submitButton: '.form-login button[type="submit"]'
+  };
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ FormLoginComponent ]
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(FormLoginComponent);
+    component = fixture.componentInstance;
+    baseElement = fixture.nativeElement;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
@carryall  In rails spec, we try to [avoid](https://www.betterspecs.org/#should) `should` word. Shall we avoid it in the JS spec also? 🤔